jwst/lib/basic_utils.py

```diff
@@ -1,5 +1,33 @@
 """General utility objects"""
 
+# Moved to stdatamodels, kept here to preserve interface
+from stdatamodels.jwst.library.basic_utils import deprecate_class, bytes2human  # noqa: F401
```

Comment:
I think this line should be removed, from https://github.com/spacetelescope/jwst/pull/8325/files